python/pyarrow/__init__.py

```diff
@@ -105,6 +105,16 @@
 
 localfs = LocalFileSystem.get_instance()
 
+# Entry point for starting the plasma store
+
+def start_plasma_store():
```

Comment:
I'd prefer to not have this as part of the official API, since it is reading arguments from sys.argv; I'll rename it to _plasma_store_entry_point, any idea if there is a file where it fits better?

If we want a function that starts the store, we can move the one from test_plasma into the pyarrow.plasma namespace.